libsel4vm/src/arch/x86/ept.c

```diff
@@ -28,13 +28,13 @@
 void print_ept_violation(vm_vcpu_t *vcpu)
 {
     /* Read linear address that guest is trying to access. */
-    unsigned int linear_address;
+    seL4_Word linear_address;
     vm_vmcs_read(vcpu->vcpu.cptr, VMX_DATA_GUEST_LINEAR_ADDRESS, &linear_address);
     printf(COLOUR_R "!!!!!!!! ALERT :: GUEST OS PAGE FAULT !!!!!!!!\n");
     printf("    Guest OS VMExit due to EPT Violation:\n");
-    printf("        Linear address 0x%x.\n", linear_address);
-    printf("        Guest-Physical address 0x%x.\n", vm_guest_exit_get_physical(vcpu->vcpu_arch.guest_state));
-    printf("        Instruction pointer 0x%x.\n", vm_guest_state_get_eip(vcpu->vcpu_arch.guest_state));
+    printf("        Linear address 0x%lx.\n", linear_address);
```

Comment:
The proper way to print a 'seL4_Word' would be using the dedicated `SEL4_PRIx_word, that adapts to the arch/plat specific format. Please don't use "lx".